mitodl/ocw-hugo-themes
# move sponsor logos to the correct location

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/127

#### What's this PR do?
In https://github.com/mitodl/ocw-hugo-themes/pull/108, we moved the corporate sponsor logos into `ocw-hugo-themes`.  These were inadvertently placed in the wrong location under the `pdfjs` folder.  This PR moves them into the correct location under `www/static/images/sponsor_logos`

#### How should this be manually tested?
 - Spin up the site using `npm start` on the `main` branch
 - Force refresh the site, you should see broken images at the bottom where the sponsor logos should be
 - Switch back to `cg/fix-sponsor-logo-location` and spin the site up again
 - The images should now appear at the bottom
